### ISSUE TITLE
Proposal to change linear-genome-view to linear-view

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#1290]
I've noticed that LinearGenomeView doesn't really match the naming scheme of the rest of our views. For example, we have CircularView not CircularGenomeView, or DotplotView not DotplotGenomesView. I propose changing it to LinearView (or at least changing the name of the package to `@jbrowse/linear-view` instead of `@jbrowse/linear-genome-view`).

If people like the current name, or if it's too ingrained to change it, that's fine with me. I just thought I'd bring up in case we wanted to change it before we publish things.

**Comment by cmdcolin** [comment]
I would say keep linear-genome-view personally

**Comment by garrettjstevens** [comment]
Another option: rename circular-view to circular-genome-view

**Comment by ihh** [comment]
I'm inclined to keep linear-genome-view because it allows user-selected genome annotation tracks (which not all the views allow), but it's hard to be objective when we've used it for so long. I suppose the logical extrapolation would be that the split-view should be called the split-genome-view and the synteny view should have "genome" in it as well, and I am not sure I like either of those options. Also the "genome" in "linear-genome-view" sort of hints at the fact that this is the "default" view of genome browsers. Again, not sure any of these are good objective arguments. It's a good discussion to have (maybe you've had it already, I just stumbled on this while browsing issues...)
